docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -63,7 +63,7 @@ <h4>{{ tab.title }}</h4>
 
 <uxd-api-properties tableTitle="Methods">
     <tr uxd-api-property name="selectTab" args="tab: number | TabComponent">
-        Programmatically select a tab. Method accepts a tab index or a tab instance as an arguement.
+        Programmatically select a tab. Method accepts a tab index or a tab instance as an arguement. The index is zero based i.e. the first tab will be 0.
```

Comment:
Argument not arguement

Reply:
I should maybe get a spell checker extension.